### ISSUE TITLE
INTERNAL ERROR: Black produced code that is not equivalent to the source.

### ISSUE THREAD

**Issue by lazydroid** [psf/black#2082]
Black went belly-up with the "INTERNAL ERROR" message when encountered a TAB character within triple-quote comment. The TAB character was NOT the leading character, it was used to align things in the table like this:
```
   """
       some text <tab> some other text
       some more text <tab> some another text
   """
```

**To Reproduce** Steps to reproduce the behavior:

1. Take this file (see the triple-comment above)
2. Run _Black_ on it without any arguments, but the file name
3. See error

**Expected behavior** Successful formatting.

**Environment (please complete the following information):**

- Version: black-20.8b1.tar.gz
- Linux/Python 3.6

**Comment by dpalmasan** [comment]
Some more details, I noticed that this happens:

```python
"""
    some text		some other text
    some more text		some another text
"""

def something():
    """
    some text		some other text
    some more text		some another text
    """
    pass
```

Only in the second case (where there is indentation for the triple quote. I also noticed that this was fixed, at least in the dev branch, as I cannot reproduce in dev mode (`pipenv shell`; `black, version 20.8b2.dev126+g9cbf1f1`). Maybe this is already fixed?

**Comment by ambv** [comment]
Yeah, this is fixed as of 21.4b0, thanks for your report.
